test/googmodule_test.ts

```diff
@@ -420,6 +420,7 @@ goog.loadedModules_['legacy.bar.baz'] = { exports: exports, type: goog.ModuleTyp
         import * as starImport from './relpath.js';
         import {namedImport, renamedFrom as renamedTo} from '../dotdot/file.js';
         export * from './exportStar.js';
+        export * as ns from './exportStarAsNs.js';
```

Comment:
nit: maybe add a new `it` block for this? This is shoehorning a lot of different functionality into a single test case. Not your fault, but let's maybe stop adding to it :-)

Reply:
Good suggestion, done! 😄